src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Scanner;
+
```

Comment:
Secondly, not using classes as recommended for the iP for Task, Todo, Deadline, Event etc.